src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -2244,6 +2244,11 @@ public async Task DeactivateUserAsync(OrganizationUser organizationUser, Guid? d
             {
                 try
                 {
+                    if (organizationUser.Status == OrganizationUserStatusType.Deactivated)
+                    {
+                        throw new BadRequestException("Already deactivated.");
```

Comment:
Do we want to just skip them instead of throw an error?

Reply:
That's basically what was happening... we still move on because the try is inside of the loop and adds validation. This was the pattern that I reused here so the entire call won't fail, it'll just queue up those errors during the update of the rest of them.